main.tf

```diff
@@ -251,7 +251,7 @@ resource "aws_lb_listener_rule" "mdw_ws" {
 
   condition {
     path_pattern {
-      values = ["/v2/websocket"]
```

Comment:
this is middleware routings not aenode

Reply:
[fd468d2](https://github.com/aeternity/terraform-aws-api-loadbalancer/pull/21/commits/fd468d201898f298efb3a62a8236734135aa8ad8)